elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -26,9 +26,39 @@ class PfeAccordion extends PFElement {
     return "pfe-accordion.json";
   }
 
-  static get cascadingAttributes() {
+  static get properties() {
     return {
-      on: "pfe-accordion-header, pfe-accordion-panel"
+      color: {
+        title: "Color",
+        type: String
+      },
+      pfeColor: {
+        // @TODO: Deprecate property
+        type: String,
+        prefix: false,
+        alias: "color"
+      },
+      disclosure: {
+        title: "Disclosure",
+        type: String,
```

Comment:
Wouldn't this be boolean?

Reply:
We're going to leave this for now. This is an opt out feature.